build.gradle

```diff
@@ -54,3 +55,27 @@ compileTestKotlin {
 task wrapper(type: Wrapper) {
     gradleVersion = '4.7'
 }
+
+detekt {
+    input = files("src/main/kotlin")
+
+    config = files("config/detekt/detekt.yml")
```

Comment:
I'd remove `detekt.yml` as it mirrors the defaults. Personally I would use the config only for overrides via `buildUponDefaultConfig = true` flag - but I don't think that is necessary at the moment.